core/demo/cli_cube_predicate.yaml

```diff
@@ -0,0 +1,19 @@
+pipeline: "BasicBatchPipeline"
+
+inputURI: "csv://core/demo/sample_cubed.csv"
+
+classifier: "predicate"
+countColumn: "count"
+meanColumn: "mean"
+stdColumn: "std"
+
+metric: "mean"
+predicate: "<="
```

Comment:
Lol it's hard to tell if the output for this query actually corresponds to "correct" output right? Can we create a new csv file that realy showcases the predicate classifier?